gen/templates/resource.go.tmpl

```diff
@@ -262,48 +284,74 @@ func (r *{{.ResourceClassName}}Resource) Configure(ctx context.Context, req reso
 func (r *{{.ResourceClassName}}Resource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
 	tflog.Debug(ctx, "Start create of resource: aci_{{.ResourceName}}")
 	
-	{{- if .HasChild}}
-	// On create retrieve information on current state prior to making any changes in order to determine child delete operations
-	var stateData *{{.ResourceClassName}}ResourceModel
-	resp.Diagnostics.Append(req.Plan.Get(ctx, &stateData)...)
-	set{{.ResourceClassName}}Id(ctx, stateData)
-	getAndSet{{.ResourceClassName}}Attributes(ctx, &resp.Diagnostics, r.client, stateData)
-	{{- end}}
```

Comment:
why removed the indentation in the template files? I see it in more locations happening.